### ISSUE TITLE
Custom Pagination CSS

### ISSUE THREAD

**Issue by freyandhy** [ashvin27/react-datatable#19]
![image](https://user-images.githubusercontent.com/12264875/68215364-10d89300-001a-11ea-95a9-0cd59bd98035.png)

Can i config the CSS for pagination ? like active class etc.
And how if i want to show not only active page ?

Maybe like some config at this component
https://www.npmjs.com/package/react-js-pagination

Thanks.

**Comment by ashvin27** [comment]
We are working on it, we will release it soon.

**Comment by ashvin27** [comment]
Hi,
We have implemented new type of pagination like you want in new release(1.3.0) please see documentation.
Also please let us know if you have any problem in same

**Comment by freyandhy** [comment]
Hi @ashvin27, thanks for the new update.

i just try it and works good, but i found some issue.

i set `next` & `previous` at `pagination`, `language` config but doesn't work. the default is always left and right arrow.

this is my config
```
language: {
  pagination: {
    previous: "Previous",
    next: "Next"
  }
}
```

i think the problem is at `lib/ADPagniation.js`

```
tags.push(_react2.default.createElement(
  Item,
  {
    key: "p0",
    className: props.isFirst ? "disabled " : "",
    onClick: props.previousPage },
  "\u25C4"
));

.
.
.

tags.push(_react2.default.createElement(
  Item,
  {
    key: "n0",
    className: props.isLast ? "disabled " : "",
    onClick: props.nextPage },
  "\u25BA"
));
```

that code still use `"\u25C4"` & `"\u25BA"`, unicode for black left & right pointing pointer

**Comment by ashvin27** [comment]
Hi @freyandhy Thanks for the update. 

We will fix it then update.

**Comment by ashvin27** [comment]
Fixed in version 1.5.0
